app/code/Magento/ReviewGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,69 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Query {
+    productReviews (
+        entity_id: Int @doc(description: "The product id to get reviews for"),
+        pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once. This attribute is optional."),
```

Comment:
Let's add sort order direction argument. It should affect how we sort by creation date. Descending should be the default value.